webpack/assets/javascripts/react_app/components/Editor/EditorHelpers.js

```diff
@@ -0,0 +1,159 @@
+import {
+  isEmpty,
+  filter,
+  find,
+  replace,
+  capitalize,
+  forEach,
+  concat,
+  sortBy,
+} from 'lodash';
+
+import ace from 'brace';
+import 'brace/ext/language_tools';
```

Comment:
[2022-01-11T09:32:31.759Z] ModuleNotFoundError: Module not found: Error: Can't resolve 'brace/ext/language_tools' in '/home/jenkins/workspace/foreman-packaging-deb-pr-test/debian/bullseye/build-foreman/foreman-3.2.0/webpack/assets/javascripts/react_app/components/Editor'

:crying_cat_face:

Reply:
Same on RPM:
```
resolve 'brace/ext/language_tools' in '/builddir/build/BUILD/foreman-3.2.0-develop/webpack/assets/javascripts/react_app/components/Editor'
```

Reply:
Seems `language_tools` is not exported from `vendor-core`?

https://github.com/theforeman/foreman-js/blob/1ee3ffa13a98fae2e839e3fb5c7cc69e00d1e503/packages/vendor-core/lib/modules.js#L75

Reply:
Yeah, it seems so :thinking:
My bad for not checking, I'll see to fixing it :)

Reply:
We really need a CI step that tries to compile stuff like prod does, without all those additional NPM packages.